classes/task/sendfailednotifications_task.php

```diff
@@ -27,6 +27,16 @@
 
 class sendfailednotifications_task extends \core\task\scheduled_task {
 
+    /**
+     * Constants
+     * Repurpose email_to_user() to send for users with just email addresses.
+     */
+    const DEFAULT_RECEIVER = -99;
+    const DEFAULT_RECEIVER_NAME = "";
+    const DEFAULT_SENDER = -99;
+    const DEFAULT_SENDER_NAME = "";
+    const DEFAULT_SENDER_EMAIL = "";
```

Comment:
this should be a valid e-mail if you don't want to send it to spam folder.
for example: no-reply@example.com

Reply:
The easier option here is using the following for the from user
`$noreplyuser = \core_user::get_noreply_user();`

Reply:
When I leave this blank the email is from noreply@localhost.com as configured in the "No-reply address" in Moodle.